src/views/v2/month/day.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * View: Month View - Day
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/day.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+// @todo: This is just for presentation purposes, while building the view.
+$day_number  = ( $week < 1 ) ? ( $day + 1 ) * ( $week + 1 ) : ( $day + 1 ) + $week * 7;
+
+$day_classes = [ 'tribe-events-calendar-month__day-date', 'tribe-common-h4' ];
```

Comment:
Desktop seems to use `desktop-h4` size, but mobile uses `mobile-h6` size. This would be a good case to define these in the `tribe-events-calendar-month__day-date` CSS instead. The headings are not yet mixins, but we should probably turn them into mixins for cases like this.

Reply:
I've added heading mixins to the latest `tribe-common-styles`, check out https://github.com/moderntribe/tribe-common/pull/1029 on how they're being used.